regnety/evalaute.py

```diff
@@ -0,0 +1,79 @@
+import tensorflow as tf
```

Comment:
Should this file have been deleted in favour of regnety/evaluate.py? It looks like they are the same, and I'm guessing this one's name was a typo

Reply:
Sorry, I was pushing from multiple endpoints. I'll check and remove that asap